Update new developer readme (#392)
hackforla/VRMS: README.md

# Volunteer Relationship Management System

[VRMS](https://www.vrms.io/) is a tool used for the engagement, support, and retention of a network of volunteers.

This project aims to help us increase engagement, descrease the time needed to onboard to projects, and improve outcomes for both volunteers and the projects they contribute to.

## Technologies

This is a Full Stack web app, built with:

- [React](https://reactjs.org/docs/getting-started.html)
- [Node](https://nodejs.org/en/)
- [Express](https://expressjs.com/en/starter/installing.html)
- [MongoDB](https://docs.mongodb.com/manual/tutorial/getting-started/)
- [Docker](https://www.docker.com/get-started)
- [Github Actions](https://github.com/features/actions)

> - [AWS](https://aws.amazon.com/)

Reviewer: Let's add:
- [Redux](https://redux.js.org/introduction/getting-started)
- [Sass](https://sass-lang.com/guide)
- [Jest](https://jestjs.io/)
- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro)

Author: 👍

# Volunteer Relationship Management System

> [VRMS](https://www.vrms.io/) is a tool used for the engagement, support, and retention of a network of volunteers.

Reviewer 2: ```suggestion
[VRMS](https://www.vrms.io/) is a tool used for the engagement, support, and retention of a network of software development volunteers.
```

> This project aims to help us increase engagement, descrease the time needed to onboard to projects, and improve outcomes for both volunteers and the projects they contribute to.

Reviewer 2: ```suggestion
This product aims to increase volunteer engagement, decrease the time needed to onboard to projects, and improve satisfaction both for volunteers and with the projects they contribute to.
```
